tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/config/CloudWorkflowConfig.scala

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.dsp.pipelines.config
+
+import org.broadinstitute.dsp.pipelines.commandline.PapiVersion.PAPIv2
+import org.broadinstitute.dsp.pipelines.commandline.{
+  CromwellEnvironment,
+  PapiVersion,
+  PipelineTestType,
+  WorkflowTestCategory
+}
+
+case class CloudWorkflowConfig(
+    category: WorkflowTestCategory = WorkflowTestCategory.Plumbing,
+    env: CromwellEnvironment = CromwellEnvironment.Dev,
+    pipeline: PipelineTestType = PipelineTestType.ExomeGermlineSingleSample,
+    truthBranch: String = "develop",
+    papiVersion: PapiVersion = PAPIv2,
```

Comment:
I *think* we can get rid of PapiVersion now.  everything should default to PAPIv2 (and you really shouldn't need to set it)

Reply:
We don't have a command line flag to set it, we are just assigning it a default value in the config here. Probably just not going to mess with it...